catalyst-toolbox/src/rewards/veterans.rs

```diff
@@ -16,6 +16,15 @@ pub struct VeteranAdvisorIncentive {
     pub reputation: u64,
 }
 
+pub struct FinalRankingWithConsensus {
```

Comment:
this does not need to be `pub`, same for fields

Reply:
fixed